src/greaseweazle/image/scp.py

```diff
@@ -110,7 +121,7 @@ def from_file(cls, name):
             dat = f.read()
 
         header = struct.unpack("<3s9BI", dat[0:16])
-        sig, _, disk_type, nr_revs, _, _, flags, _, single_sided, _, _ = header
+        sig, version, disk_type, nr_revs, _, _, flags, _, single_sided, _, _ = header
```

Comment:
Why?

Reply:
Ah I reverted this later - will remove the patch for this.